pkg/bmc/access_test.go

```diff
@@ -372,6 +372,18 @@ func TestStaticDriverInfo(t *testing.T) {
 			vendor: "",
 		},
 
+		{
+			Scenario: "idrac redfish",
+			input:    "idrac+redfish://192.168.122.1",
```

Comment:
It's `idrac-redfish` in the docs (not `idrac+redfish`). I think `-` is correct for consistency with #411. So far we've only used `+` for `+http` and `+https`.